govcd/metadata.go

```diff
@@ -247,6 +251,109 @@ func (mediaItem *MediaItem) DeleteMetadata(key string) error {
 
 // DeleteMetadataAsync() function calls private function deleteMetadata() with mediaItem.client and mediaItem.MediaItem.HREF
 // which deletes metadata depending on key provided as input from media item.
+// Deprecated: Use MediaRecord.DeleteMetadataAsync
 func (mediaItem *MediaItem) DeleteMetadataAsync(key string) (Task, error) {
 	return deleteMetadata(mediaItem.vdc.client, key, mediaItem.MediaItem.HREF)
 }
+
+// GetMetadata() function calls private function getMetadata() with mediaItem.client and mediaItem.MediaItem.HREF
```

Comment:
`mediaItem*` should be `mediaRecord`

Same for the other functions below.

Reply:
updated